android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -179,7 +188,17 @@ avifResult DecodeNextImage(JNIEnv* const env, AvifDecoderWrapper* const decoder,
     LOGE("Failed to decode AVIF image. Status: %d", res);
     return res;
   }
-  return AvifImageToBitmap(env, decoder, bitmap);
+  return AvifImageToBitmap(env, decoder, bitmap, false);
+}
+
+avifResult DecodeNextImage(JNIEnv* const env, AvifDecoderWrapper* const decoder,
```

Comment:
Duplicating this function is not necessary. You can simply pass true/false from each caller instead.

Reply:
yes